css/style.css

```
@@ -375,6 +471,19 @@ img {
   line-height: 21px;
   background-color: var(--blue);
   background: url(../img/filter-background.jpg) no-repeat;
+  padding-top: 27px;
+  padding-bottom: 31px;
+  padding-left: 73px;
+  padding-right: 73px;
```

При указании всех 4-х свойств можно пользоваться сокращенной записью, которую ты уже используешь в нескольких местах:
```suggestion
  padding: 27px 73px 31px 73px;
```

или даже так:
```suggestion
  padding-top: 27px 73px 31px;
```